### DISCUSSION TITLE
Analyzing results and plotting graphs in OMNeT++ for comparing two or more mechanisms

### DISCUSSION THREAD

**Discussion by soreettihussein** [omnetpp/omnetpp#1149] [Q&A]
Hello everyone,

I am new to using the OMNeT++ simulator and currently in the process of familiarizing myself with the examples and showcases provided by the inet framework. While exploring, I have encountered difficulties in understanding analyzing results and plotting graphs in OMNeT++ for comparing two or more mechanisms from .vec and .sca under result directory, such as measuring end-to-end delay (as illustrated in the example here: https://inet.omnetpp.org/docs/showcases/tsn/framepreemption/doc/index.html).

I am running OMNeT++ on Windows 11.

I would greatly appreciate any assistance or guidance from experienced users who could help me understand the techniques for comparing multiple mechanisms effectively.
Thank you!

**Comment by torokati44** [comment]
> I have encountered difficulties in understanding analyzing results and plotting graphs

It's neigh impossible to help you without you first explaining what exactly you're trying to achieve, what have you tried already, and why it hasn't worked out for you.

**Reply by soreettihussein** [reply]
please can you give me guidance’s (explanation) how was the comparison of mean **end-to-end delay** from  .vec and .sca under result directory of FifoQueueing, PriorityQueueing and Frame Preemption graph was plotted. Illustrated in the example here: https://inet.omnetpp.org/docs/showcases/tsn/framepreemption/doc/index.html). I am using OMNeT++ 6.0.2 on Windows 11.
Thank you in advances!!!
